website/server/controllers/api-v3/tasks.js

```diff
@@ -241,6 +254,17 @@ api.createChallengeTasks = {
 
     // If adding tasks to a challenge -> sync users
     if (challenge) challenge.addTasks(tasks);
+
+    tasks.forEach((task) => {
+      res.analytics.track('create task', {
```

Comment:
Do you need a 7 days old check here?

Reply:
I'm thinking because adding challenge tasks is a much less frequent flow than users adding their own tasks, leaving them on would be a negligible event volume and wouldn't need such a cutoff.

That said, maybe adding tasks to challenges isn't useful information in addition to creating the challenge in the first place?

Reply:
Sounds good to me! Just making sure. I would like to keep these and see. We can always change.